Dockerfile

```diff
@@ -18,13 +18,13 @@ COPY cmd/    cmd/
 
 # Build
 RUN CGO_ENABLED=0 GOOS=linux \
-		go build \
-            -mod readonly \
-			-ldflags "$GO_LDFLAGS" -tags="$GO_TAGS" -a \
-			-o elastic-operator github.com/elastic/cloud-on-k8s/cmd
+      go build \
+      -mod readonly \
+      -ldflags "$GO_LDFLAGS" -tags="$GO_TAGS" -a \
+      -o elastic-operator github.com/elastic/cloud-on-k8s/cmd
 
 # Copy the operator binary into a lighter image
-FROM registry.access.redhat.com/ubi8/ubi-minimal:8.5
+FROM scratch
```

Comment:
Should this be `distroless` instead? We used to use that in the past. AFAIK one of the advantages is that it comes with CA certificates and supports tmp (not sure if we need the latter)

Reply:
I can move this to distroless.  I had run through all e2e tests using scratch, and `-ubi` builds and hadn't hit a single failure.

Reply:
I am trying to think of scenarios where the operator actually makes HTTP requests that would require root CA certs. The only one I can think of now is the external secretes feature we added recently. The operator makes a request to the linked Elastic Stack applications to determine their version, which would require those certs. We don't test this setup in e2e tests.  @thbkrkr may have more insights here. 

But I guess `distroless` puts us on the safe side wrt that.

Reply:
Yes, we are in a better place using `distroless` for the reasons you outlined.